spec/models/spree/calculator/flat_percent_item_total_spec.rb

```diff
@@ -4,13 +4,20 @@
   let(:calculator) { Spree::Calculator::FlatPercentItemTotal.new }
   let(:line_item) { instance_double(Spree::LineItem, amount: 10) }
 
-  before { calculator.stub :preferred_flat_percent => 10 }
+  before { calculator.stub preferred_flat_percent: 10 }
 
-  it "should compute amount correctly for a single line item" do
+  it "computes amount correctly for a single line item" do
     calculator.compute(line_item).should == 1.0
   end
 
   context "extends LocalizedNumber" do
     it_behaves_like "a model using the LocalizedNumber module", [:preferred_flat_percent]
   end
+
+  it "computes amount correctly for a given Spree::Stock::Package" do
```

Comment:
Should it mention [Stock::Package](https://github.com/openfoodfoundation/openfoodnetwork/blob/2-0-stable/app/models/stock/package.rb) instead?

Reply:
yes! Fixed. Thanks!